src/qrl/core/p2p/p2pTxManagement.py

```diff
@@ -10,6 +10,12 @@
 
 
 class P2PTxManagement(P2PBaseObserver):
+    """
+    P2PTxManagement is simply a collection of functions that are called when
```

Comment:
Comment for all the functions inside P2PTxManagement that has been registered by Observable.register, will be more helpful.